modules/gdscript/gdscript_functions.cpp

```diff
@@ -1217,8 +1217,8 @@ void GDScriptFunctions::call(Function p_func, const Variant **p_args, int p_arg_
 					return;
 				}
 			}
-
-			r_ret = gdscr->_new(NULL, 0, r_error);
+			const int skip_initializer = -1;
+			r_ret = gdscr->_new(NULL, skip_initializer, r_error);
```

Comment:
```c++
static const SKIP_INITIALIZER  = -1; // I prefer an enum or macro more
// or
r_ret = gdscr->_new(NULL, -1/*skip initializer*/, r_error);
```

Reply:
I like the inline comment too but I'm not sure about the specific style requirements...